docs/docker.md

```diff
@@ -19,8 +19,11 @@ Additionally, there the following images, similar to the above:
 - `ghcr.io/ggerganov/llama.cpp:full-rocm`: Same as `full` but compiled with ROCm support. (platforms: `linux/amd64`, `linux/arm64`)
 - `ghcr.io/ggerganov/llama.cpp:light-rocm`: Same as `light` but compiled with ROCm support. (platforms: `linux/amd64`, `linux/arm64`)
 - `ghcr.io/ggerganov/llama.cpp:server-rocm`: Same as `server` but compiled with ROCm support. (platforms: `linux/amd64`, `linux/arm64`)
+- `ghcr.io/ggerganov/llama.cpp:full-musa`: Same as `full` but compiled with MUSA support. (platforms: `linux/amd64`)
+- `ghcr.io/ggerganov/llama.cpp:light-musa`: Same as `light` but compiled with MUSA support. (platforms: `linux/amd64`)
+- `ghcr.io/ggerganov/llama.cpp:server-musa`: Same as `server` but compiled with MUSA support. (platforms: `linux/amd64`)
```

Comment:
Note that for the images to be uploaded to ghcr.io they need to be added to `docker.yml`.

Reply:
Thanks for the hints! I plan to add them in a separate PR, as the final image of `musa:3.1.0-devel/runtime` will be available after our National Day holiday.

Reply:
Please see the draft PR: https://github.com/ggerganov/llama.cpp/pull/9691

Reply:
I can merge this now if you want, but until this is added to the ci, these images are not going to be available on ghcr.io for the users trying to follow instructions from this document.

Reply:
I understand your concern. It seems combining the two PRs might be a better approach. Let's keep this one open until the base images are ready and published.

I’ll convert it to a draft. Let’s revisit once the base images are ready.

Reply:
> Please see the draft PR: #9691

Closed.